doc/pages/user/060-using_triggers.md

```diff
@@ -5,7 +5,8 @@ data. More details on Triggers can be found in the [Architecture Documentation](
 
 Astarte allows you to install and delete Triggers dynamically through its clients. Upon installation
 or deletion, changes to the Trigger infrastructure might take some time to propagate, and some devices
-might pick up changes at a later time. If a Trigger shows as installed, it will eventually be loaded.
+might pick up changes at a later time. If a Trigger shows as installed, it will eventually be
+loaded. In the current implementation of Astarte, the interval for trigger reloading is 10 minutes.
```

Comment:
`In the current implementation of Astarte` doesn't really make sense, the documentation is versioned. Just omit that part.